scripts/profiler/plot_nordic.py

```diff
@@ -288,43 +294,53 @@ def button_press_event(self, event):
             if selected_event is None:
                 return
 
+            if self.processed_data.tracking_execution:
+                event_submit = selected_event.submit
+            else:
+                event_submit = selected_event
+
             self.draw_state.selected_event_submit = self.draw_state.ax.plot(
-                selected_event.submit.timestamp,
-                selected_event.submit.type_id,
+                event_submit.timestamp,
+                event_submit.type_id,
                 markersize=2*self.draw_state.event_submit_markersize,
                 color='g',
                 marker='o',
                 linestyle=' ')
-            self.draw_state.selected_event_processing = matplotlib.patches.Rectangle(
-                (selected_event.start.timestamp,
-                 selected_event.submit.type_id -
-                 self.draw_state.event_processing_rect_height),
-                selected_event.end.timestamp -
-                selected_event.start.timestamp,
-                2*self.draw_state.event_processing_rect_height,
-                color='g')
-            self.draw_state.ax.add_artist(
-            self.draw_state.selected_event_processing)
+
+            if self.processed_data.tracking_execution:
+                self.draw_state.selected_event_processing = matplotlib.patches.Rectangle(
+                    (selected_event.start.timestamp,
+                    selected_event.submit.type_id -
+                    self.draw_state.event_processing_rect_height),
+                    selected_event.end.timestamp -
+                    selected_event.start.timestamp,
+                    2*self.draw_state.event_processing_rect_height,
+                    color='g')
+                self.draw_state.ax.add_artist(
+                self.draw_state.selected_event_processing)
```

Comment:
Argument of `add_artist()` should be intended if it goes to a new line.

Reply:
done